config/database.yml

```diff
@@ -0,0 +1,89 @@
+# PostgreSQL. Versions 9.1 and up are supported.
+#
+# Install the pg driver:
+#   gem install pg
+# On OS X with Homebrew:
+#   gem install pg -- --with-pg-config=/usr/local/bin/pg_config
+# On OS X with MacPorts:
+#   gem install pg -- --with-pg-config=/opt/local/lib/postgresql84/bin/pg_config
+# On Windows:
+#   gem install pg
+#       Choose the win32 build.
+#       Install PostgreSQL and put its /bin directory on your path.
+#
+# Configure Using Gemfile
+# gem 'pg'
+#
+default: &default
+  adapter: postgresql
+  encoding: unicode
+  # For details on connection pooling, see Rails configuration guide
+  # http://guides.rubyonrails.org/configuring.html#database-pooling
+  host: localhost
+  port: 5432
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  username: 'wbooks-api'
```

Comment:
Remember this fields should search an env variable: `username: <%= ENV.fetch('DB_USERNAME', 'wbooks-api') %>`

Reply:
Same for: host, port, and password

Reply:
Okay, I'll change all these lines, but should I leave the specification of ENV variables for later? I thought I was told I would be requested to do this task (defining ENV variables and .env file) later in another card, but I'm not so sure now.

Reply:
Add it here, in a future task you will have to manage sensible data of our application and you would exploded better the ENV potential. But, as you used ENV before in this PR, change the database config to use ENV var